packages/components/tooltip/src/hooks/__tests__/useClickOutside.spec.tsx

```diff
@@ -0,0 +1,56 @@
+import { renderHook } from '../../../../../../tests/test-utils';
+import { useClickOutside } from '../useClickOutside';
+import { fireEvent } from '@testing-library/react';
+
+describe('useClickOutside', () => {
+  it('should trigger the handler when clicking outside the element', () => {
+    const handler = jest.fn();
+    const ref = { current: document.createElement('div') };
+
+    renderHook(() => useClickOutside(ref, handler));
+
+    document.body.appendChild(ref.current);
+
+    fireEvent.mouseDown(document.body);
+
+    expect(handler).toHaveBeenCalledTimes(1);
+
+    document.body.removeChild(ref.current);
```

Comment:
This action could be set in 'afterAll' function outside each test at the beginning.

```suggestion
    afterAll(() => document.body.removeChild(ref.current);)
```